mediorum/server/serve_upload.go

```diff
@@ -114,6 +119,25 @@ func (ss *MediorumServer) postUpload(c echo.Context) error {
 	return c.JSON(200, uploads)
 }
 
+func (ss *MediorumServer) getJobResult(c echo.Context) error {
```

Comment:
perhaps a better name would be something like: `getV1CidBlob`.

To me the `job` refers to the database record that tracks progress and whatnot... and `blob` refers to the actual goods.

Also realizing that the `get` prefix is kind of confusing here... maybe we change that convention to be more explicit like `serveXyzGET` and `serveAbcPOST` or something... but we can do that later.

Reply:
makes sense, I changed it to `getV1CIDBlob`